README.md

```diff
@@ -74,8 +74,7 @@ If you have a Snyk token, add that as an environment variable:
  - name: _HIGHEST_ALLOWED_VULN_SEVERITY_, Possible values: [`LOW`,`MEDIUM`,`HIGH`,`CRITICAL`], Description: will determine what is the highest allowed Severity of a vulnerability found for a given package/dependency in the analysis, for the analysis to be considered Successful(RC=0) and not Vulnerable(RC=2), Default value is `MEDIUM`
 
 #### Python Pipeline Configuration
- For python PIP Packages, you can bypass using the python and pip binaries during the invocation of the analysis, and just use them elsewhere in the pipeline job (another stage, or even another agent/node),
- The Idea is to bring maximum flexibility with the python and pip versions, so you won't enforce the user to install sometimes a different python and pip versions just to adapt it to the exact requirements.txt' packages' versions, as python pip is very sensitive to versioning (for each python and pip version , there is only a limited range of supported versions for every package).
+ For Python PIP packages, you can bypass using the Python and PIP binaries during the invocation of the analysis. You can also use them elsewhere in your pipeline jobs, such as another stage, or another agent or node. The Idea is to bring maximum flexibility with the python and pip versions, so you won't enforce the user to install sometimes a different python and pip versions just to adapt it to the exact requirements.txt' packages' versions, as python pip is very sensitive to versioning (for each python and pip version , there is only a limited range of supported versions for every package).
```

Comment:
We want to capitalize proper nouns, such as, `python` --> `Python`.  Also, we should be consistent with acronyms, `pip` --> `PIP`.

`The Idea ...` --> `The idea ...`

We should not use contractions in technical documentation. For example, `... so you won't enforce ...` --> `... so you do not enforce ...`.

Not sure why single quotes are used here:

`... to the exact requirements.txt' packages' versions ...`

Maybe this is just a typo, but I think the first single quote can be removed, and you can leave the other single quote, because `packages'` is being used in the possessive (as owning the versions).

Also, in technical documentation you should not have parenthetical statements.

The third sentence is a run-on sentence and a rewrite should be considered.

Here is my suggestion for this line:
For Python PIP packages, you can use the specific Python and PIP binaries during the invocation of the analysis.
You can also specify these binaries elsewhere in your pipeline jobs, such as a stage environment, or another agent or node.
Red Hat Dependency Analytics gives you maximum flexibility with the Python and PIP versions.
You do not have to enforce the user to install different Python and PIP versions just to adapt it to the exact `requirements.txt` list of package versions.
Python is very sensitive to versioning, for each Python version, there is a limited range of supported versions for a package.

Reply:
@ritz303 Thanks, Applied these changes.